ProcessLib/ProcessType.h

```diff
@@ -10,12 +10,12 @@
  * Created on December 14, 2016, 3:13 PM
  */
 
-#ifndef OGS_PROCESS_TYPE_H
-#define OGS_PROCESS_TYPE_H
+
+#pragma once
 
 namespace ProcessLib
 {
-enum class ProcessType
```

Comment:
This change will lead to integer ids of the process type being printed on error. I'm not happy with that.